.github/configuration/upgrade-test-configuration.yml

```diff
@@ -122,6 +122,18 @@ upgrade-version: [{
     }
   ],
 },
+{
+  upgrade-path: [
+    {
+      version: '14.19',
```

Comment:
Let's update the existing 14.18 -> latest to 14.19 -> latest so as to not unnecessary create too many github workflows.